src/Microsoft.Azure.WebJobs.Host/WebJobs.Host.csproj

```diff
@@ -952,6 +1060,7 @@
       <Link>Properties\PublicKey.snk</Link>
     </None>
     <None Include="app.config" />
+    <None Include="ApplicationInsights.config" />
```

Comment:
You do not need this one. Please remove to avoid possible strange bugs of unexpected telemetry configuration.